wallets/react/src/types.ts

```diff
@@ -20,8 +21,6 @@ export type ConnectResult = {
   provider: any;
 };
 
-export type Providers = { [type in WalletType]?: any };
```

Comment:
It's part of the `wallets/react` API and should be kept here. Please revert this change.

Reply:
fixed.